docs/aspnetcore-webapi.md

```diff
@@ -0,0 +1,55 @@
+[<img src="https://devforum.okta.com/uploads/oktadev/original/1X/bf54a16b5fda189e4ad2706fb57cbb7a1e5b8deb.png" align="right" width="256px"/>](https://devforum.okta.com/)
+
+[![Support](https://img.shields.io/badge/support-Developer%20Forum-blue.svg)](https://devforum.okta.com/)
+
+## Installing the package
+
+You can install the package by using the NuGet Package Explorer to search for [Okta.AspNetCore](https://nuget.org/packages/Okta.AspNetCore).
+
+Or, you can use the `dotnet` command:
+
+```
+dotnet add package Okta.AspNetCore
+```
+
+# Usage example
+
+Okta plugs into your OWIN Startup class with the `UseOktaWebApi()` method:
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddAuthentication(options =>
+    {
+        options.DefaultAuthenticateScheme = OktaDefaults.ApiAuthenticationScheme;
+        options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+        options.DefaultSignInScheme = JwtBearerDefaults.AuthenticationScheme;
```

Comment:
These need to all be the same, like: https://github.com/okta/samples-aspnetcore/blob/master/resource-server/okta-aspnetcore-webapi-example/Startup.cs#L24-L26

Reply:
Yep, I was thinking on adding an issue to make the changes/improvements we have introduced in our new samples in the old ones. Sounds good?

Reply:
Sounds good to me 👍